q2cli/handlers.py

```diff
@@ -301,7 +319,7 @@ def get_value(self, arguments, fallback=None):
 
 
 class MetadataCategoryHandler(Handler):
-    def __init__(self, name, default=NoDefault):
+    def __init__(self, name, default=NoDefault, description=None):
```

Comment:
Nice catch! I think it definitely should, as I passed it along for all the other ones. Missed one I guess...